tests/pytests/test_multithread.py

```diff
@@ -56,32 +57,24 @@ def testSimpleBuffer():
 def testMultipleBlocksBuffer():
     CreateAndSearchSortBy(docs_count = 2500)
 
-
-@skip(asan=True)
-def test_invalid_MT_MODE_FULL_config():
-    try:
-        env = initEnv(moduleArgs='WORKER_THREADS 0 MT_MODE MT_MODE_FULL')
-        prefix = '_' if env.isCluster() else ''
-        env.cmd(f"{prefix}ft.config", "get", "WORKER_THREADS")
-        env.assertFalse(True)   # We shouldn't get here
-    except Exception:
-        # Create dummy env to collect exit gracefully.
-        env = Env()
-        pass
-
-
-@skip(asan=True)
-def test_invalid_MT_MODE_ONLY_ON_OPERATIONS_config():
-    # Invalid 0 worker threads with MT_MODE_ONLY_ON_OPERATIONS.
-    try:
-        env = initEnv(moduleArgs='WORKER_THREADS 0 MT_MODE MT_MODE_ONLY_ON_OPERATIONS')
-        prefix = '_' if env.isCluster() else ''
-        env.cmd(f"{prefix}ft.config", "get", "WORKER_THREADS")
-        env.assertFalse(True)   # We shouldn't get here
-    except Exception:
-        # Create dummy env to collect exit gracefully.
-        env = Env()
-        pass
+# Skipping for cluster as we only test for loading the module to the shard.
+# Sanitizer is not tested due to a bug in redis - onFLush callback is called even though module loading
+# failed, causing access to an invalid memory that was freed (module context).
+@skip(cluster=True, asan=True)
+def test_invalid_mt_config_combinations(env):
+    if not MT_BUILD:
+        raise SkipTest('MT_BUILD is not set')
```

Comment:
you can move this to the skip, no?
```suggestion
@skip(cluster=True, asan=True, noWorkers=True)
def test_invalid_mt_config_combinations(env):
```